sdk/purview/purview-datamap-rest/README.md

```diff
@@ -0,0 +1,62 @@
+# Azure PurviewDataMap REST client library for JavaScript
+
+Purview Data Map Service is a fully managed cloud service whose users can
+discover the data sources they need and understand the data sources they find.
+At the same time, Data Map helps organizations get more value from their
+existing investments. This spec defines REST API of Purview Data Map Service.
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-datamap-rest)
+- [Package (NPM)](https://www.npmjs.com/package/@azure-rest/purview-datamap)
+- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure-rest/purview-datamap?view=azure-node-preview)
+- [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-datamap-rest/samples)
+
+## Getting started
+
+### Currently supported environments
+
+- LTS versions of Node.js
+
+### Prerequisites
+
+- You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this package.
+
+### Install the `@azure-rest/purview-datamap` package
+
+Install the Azure PurviewDataMap REST client REST client library for JavaScript with `npm`:
+
+```bash
+npm install @azure-rest/purview-datamap
+```
+
+### Create and authenticate a `PurviewDataMapClient`
+
+To use an [Azure Active Directory (AAD) token credential](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-with-a-pre-fetched-access-token),
+provide an instance of the desired credential type obtained from the
+[@azure/identity](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) library.
+
+To authenticate with AAD, you must first `npm` install [`@azure/identity`](https://www.npmjs.com/package/@azure/identity) 
+
+After setup, you can choose which type of [credential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) from `@azure/identity` to use.
+As an example, [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#defaultazurecredential)
+can be used to authenticate the client.
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+
```

Comment:
Could you improve your readme with samples and also add your samples ?
https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/steps-after-generations.md#improve-readmemd-document
https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/steps-after-generations.md#how-to-write-samples